Business Social Network/AppModules/Search/views/searchResults.html

```diff
@@ -1,66 +1,65 @@
-<html>
-
-<div class = "container"
-  ng-controller = "SearchBoxController"
+<div class = "container" ng-controller = "SearchResultsController"
   ng-show = "searchTerm.length > 0">
 
+<h1 ng-if = "searchResults.length != 1">
+  Your search for "{{searchTerm}}" returned {{searchResults.length}} results
+</h1>
 
-  <h1 ng-if = "searchResults.length != 1">
-    Your search for "{{searchTerm}}" returned {{searchResults.length}} results
-  </h1>
+<h1 ng-if = "searchResults.length == 1">
+  Your search for "{{searchTerm}}" returned {{searchResults.length}} result
+</h1>
 
-  <h1 ng-if = "searchResults.length == 1">
-    Your search for "{{searchTerm}}" returned {{searchResults.length}} result
-  </h1>
+<br /> <br />
 
-  <br /> <br />
+<table class = "table" >
 
-  <table class = "table" >
+  <thead>
 
-    <thead>
+    <tr>
+      <th>Product Name</th>
+      <th>Price</th>
+      <th>Seller</th>
+      <th>Reviews and Ratings</th>
+      <th>Product Tags</th>
+      <th>Category</th>
+    </tr>
 
-      <tr>
-        <th>Product Name</th>
-        <th>Price</th>
-        <th>Seller</th>
-        <th>Reviews and Ratings</th>
-        <th>Product Tags</th>
-      </tr>
 
+    <tr ng-repeat = "product in searchResults">
 
-      <tr ng-repeat = "product in searchResults">
+      <td>
+        <a href = "">{{product.name}}</a>
+      </td>
 
-        <td>
-          <a href = "">{{product.name}}</a>
-        </td>
+      <td>
+        {{product.price | currency: "&euro;"}}
+      </td>
 
-        <td>
-          {{product.price | currency: "&euro;"}}
-        </td>
+      <td>
+        <a href = " ">{{product.seller}}
+        </a>
+      </td>
 
-        <td>
-          <a href = " ">{{product.seller}}
-          </a>
-        </td>
+      <td>
+        {{product.review}}
+      </td>
 
-        <td>
-          {{product.review}}
-        </td>
+      <td>
+        <li ng-repeat = "tag in product.tags">
+          {{tag}}
```

Comment:
@Swailem95 
please use ng-bind instead of {{}} as this will not always work 
http://stackoverflow.com/a/16126174